common/components/intrinsics.hpp.inc

```diff
@@ -34,21 +34,21 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  * @internal
  * Returns the number of set bits in the given mask.
  */
-__device__ int popcnt(uint32 mask) { return __popc(mask); }
+inline __device__ int popcnt(uint32 mask) { return __popc(mask); }
 
 /** @copydoc popcnt */
-__device__ int popcnt(uint64 mask) { return __popcll(mask); }
+inline __device__ int popcnt(uint64 mask) { return __popcll(mask); }
 
 
 /**
  * @internal
  * Returns the (1-based!) index of the first set bit in the given mask,
  * starting from the least significant bit.
  */
-__device__ int ffs(uint32 mask) { return __ffs(mask); }
+inline __device__ int ffs(uint32 mask) { return __ffs(mask); }
```

Comment:
Usually, we use `__forceinline__` for inlining in GPU kernels (at least we always did it in CUDA and we currently use it in pretty much all `common` kernels like in `atomic` or `cooperative_groups`).